NEWS.md

```diff
@@ -20,7 +20,6 @@ This release is in development and not yet ready for production use.
 
 ## Package
 
-- Fixed some typos in `single-timeseries-rt-estimation.Rmd`. The `WORDLIST` used by spelling has also been updated to eliminate false positives. Future typos will now generate an error in the continuous integration check so that we can catch them as early as possible. See #341 by @Bisaloo and reviewed by @seabbs.
```

Comment:
why are we removing this? Is it a duplicate?

Reply:
You tell me :D
https://github.com/epinowcast/epinowcast/commit/f8e929140b51aa67e8d9b230fc6b26c9024e819c

Reply:
Weird!